ios/RNWebrtcVad.m

```diff
@@ -66,31 +67,29 @@ - (void) processSampleData:(NSData *)data
     // Google recommends sending samples (in 10ms, 20, or 30ms) chunk.
     // See: https://github.com/TeamGuilded/react-native-webrtc-vad/blob/master/webrtc/common_audio/vad/include/webrtc_vad.h#L75
 
-    const double sampleLength = 0.02;
+    const double sampleLengthMs = 0.02;
 
-    cumulativeProcessedSampleLength += sampleLength;
-    int chunkSize = sampleLength /* seconds/chunk */ * sampleRate * 2 /* bytes/sample */ ; /* bytes/chunk */
-
-    if ([self.audioData length] >= chunkSize) {
-#ifdef DEBUG
-        NSLog(@"SENDING = %u", [self.audioData length]);
-#endif
+    cumulativeProcessedSampleLengthMs += sampleLengthMs;
+    int chunkSizeBytes = sampleLengthMs /* seconds/chunk */ * sampleRate * 2 /* bytes/sample */ ; /* bytes/chunk */
 
+    if ([self.audioData length] >= chunkSizeBytes) {
+        // Convert to short pointer
         const int16_t* audioSample = (const int16_t*) [self.audioData bytes];
 
-        int isVoice = [voiceDetector isVoice:audioSample sample_rate:sampleRate length:chunkSize/2];
+        int isVoice = [voiceDetector isVoice:audioSample sample_rate:sampleRate length:chunkSizeBytes/2];
 
-#ifdef DEBUG
-        NSLog(@"Detected Voice %d", isVoice);
-#endif
 
         // Clear audio buffer
         [self.audioData setLength:0];
 
         // Sends updates ~140ms apart back to listeners
         const double eventInterval = 0.140;
```

Comment:
why is this specifically 140ms? can you expand on this in the comment?

Reply:
Good question, will upadte comment. This was chosen from some basic testing/tuning. At 20ms samples, we didn't wanna be sending events over the react native bridge so often, as it's too frequent/not useful. If we made it much longer (>=200ms) the delay of the speaking would be quite pronounced to the user. So 140ms was the nice medium